Dockerfile

```
@@ -8,7 +8,11 @@ COPY . ./
 RUN cd cmd/analyticproxy && go build; cd -; \
     cd cmd/medifor && go build; cd -
 
-FROM ubuntu:18.04
+FROM python:3.8-slim-buster
+COPY --from=builder /go/python /app/bin/
+RUN pip install -U pip
+RUN pip install -r requirements.txt
```

These should probably be on the same RUN command to limit image proliferation, and instead of installing requirements.txt, we should install the directory itself (getting setup.py invoked properly):

```suggestion
RUN pip install -U pip && pip install /src/medifor
```